packages/noya-state/src/selectors/themeSelectors.ts

```diff
@@ -62,3 +73,62 @@ export function groupThemeComponents<
 
   return array;
 }
+
+export function setComponentName<T extends ComponentsTypes>(
+  ids: string[],
+  name: string,
+  array: T[],
+) {
+  array.forEach((object: ComponentsTypes) => {
+    if (!ids.includes(object.do_objectID)) return;
+
+    const group = delimitedPath.dirname(object.name);
+    object.name = delimitedPath.join([group, name]);
+  });
+
+  return array;
```

Comment:
I would probably remove the return here, since it's uncommon to return the same object we pass as an argument